cockpit/src/subscriptions-view.jsx

```diff
@@ -257,22 +316,58 @@ class SubscriptionStatus extends React.Component {
             );
         }
 
-        let label;
+        let status_text;
         let action;
+        let org_name;
+        let sca_mode;
+        let status_element;
 
         if (this.props.status === 'unknown') {
-            label = _("Not registered");
+            status_text = _("Not registered");
             action = (
                 <Button onClick={this.handleRegisterSystem}>{_("Register")}</Button>
             );
         } else {
             const isUnregistering = (this.props.status === "unregistering");
-            label = this.props.status_msg;
+            status_text = this.props.status_msg;
             action = (
                 <Button isDisabled={isUnregistering} isLoading={isUnregistering}
                               onClick={this.handleUnregisterSystem}>{isUnregistering ? _("Unregistering"): _("Unregister")}</Button>
             );
         }
+
+        sca_mode = false;
+        if (this.props.org === undefined) {
+            org_name = '';
+        } else {
+            // Organization name
+            if ('displayName' in this.props.org) {
+                org_name = this.props.org['displayName'];
+            } else {
+                org_name = '';
+            }
+            // SCA mode tooltip
+            if ('contentAccessMode' in this.props.org) {
+                if (this.props.org['contentAccessMode'] === 'org_environment') {
+                    sca_mode = true;
+                }
+            }
+        }
+
+        // Display tooltip over "Disabled" with simple explanation, when SCA mode is used
+        if (sca_mode) {
+            const status_tooltip = (
+                <div>{_("Content Access Mode is set to Simple Content Access. This host has access to content, regardless of subscription status.")}</div>
+            );
+            status_element = (
+                <Tooltip content={status_tooltip} isContentLeftAligned='true' position='right'>
+                    <span>{status_text}</span>
```

Comment:
What is the purpose of this <span>? Feels redundant.

Reply:
When SCA mode is used, then it would be nice to print some explanation, why "Disabled" is displayed for status. We print similar explanation for CLI tool `subscription-manager status`, when SCA is used. Thus it would be good to have some functional parity for cockpit plugin. When SCA mode is not used, then there is no need to print any explanation.

Not sure if it is correct approach from UX perspective. I feel that we should emphasize that there is some tooltip explanation, when status "Disabled" (should it be link? CCS style: 'border-bottom: dashed 1px'? anything else?).

Reply:
I meant something else - resolved on IRC.